### DISCUSSION TITLE
请问有什么快捷方法，可以获取视频里指定时间范围的音频数据吗？

### DISCUSSION THREAD

**Discussion by asasas234** [WebAV-Tech/WebAV#128] [Q&A]
例如，当前视频播放的进度是第5秒，我想指定起始时间是第10秒，结束时间是第15秒，获取这5秒的音频数据，Float32Array.

请问webAV 有什么简便的方式可以实现这个需求嘛，我找了下网站的demo没找到太合适的

**Comment by hughfenghen** [comment]
参考 #124 

`IClip.split` 分割两次可得到指定范围的数据

**Comment by asasas234** [comment]
感谢，我下午试试
